.laminas-ci.json

```diff
@@ -5,6 +5,7 @@
     "ignore_php_platform_requirements": {
         "8.1": false,
         "8.2": false,
-        "8.3": true
+        "8.3": true,
+        "8.4": true
```

Comment:
Please remove all versions except the last one (8.4).

Reply:
All previous versions have been removed, leaving only the latest one (8.4)